pages/reviewer/ReviewerHandler.php

```diff
@@ -116,7 +123,8 @@ protected function _validateAccessKey($request)
         // Register the user object in the session
         $reason = null;
         if (Validation::registerUserSession($user, $reason)) {
-            $this->submission = $reviewerSubmission;
+            // TODO:: WHAT ARE THOSE?
```

Comment:
Can you clarify your question?

Reply:
It was my note in order to check what the `Validation::registerUserSession($user, $reason)` and then check what it means for the `$this->submission` not to be initialised.

Removed it